src/utils/messages.ts

```diff
@@ -0,0 +1,6 @@
+const MESSAGES = {
```

Comment:
Our code convention for enum-like consts is: `PascalCase.SCREAMING_SNAKE_CASE` so please rename to `Messages`.